Update blacksmith environment variable references

xaque208 opened the pull request.

smortex commented:
:wave: Won't this break the module publication of your other modules?

Or maybe we can't set variables for all repos for user repos the same way we can do this with organizations and the variables must be repeated for each module (I can't find Settings / Secrets for my user account)?  I would suggest keeping `PUPPET_FORGE_USERNAME` and `PUPPET_FORGE_API_KEY`to match what modulesync wants to manage by default, but if for some reason it is not possible, no strong objection to using a different name :-)

xaque208 commented:
Ah I see now that you mention it.  I've updated the keys and added the management to some terraform.  `v1.7.2` is on the Forge now. https://forge.puppet.com/modules/zleslie/ssh

Hopefully that will smooth out the release.
